crates/runtime/src/dataaccelerator/sqlite.rs

```diff
@@ -63,7 +74,11 @@ impl SqliteAccelerator {
         if !dataset.is_file_accelerated() {
             return None;
         }
-        let acceleration = dataset.acceleration.as_ref()?;
+        let mut acceleration = dataset.acceleration.clone()?;
```

Comment:
Maybe consider cloning the acceleration param instead of the whole acceleration object here. What's finally used in `sql_file_path` function is acceleration param, and cloning the whole acceleartion object can be expensive.